package.sh

```diff
@@ -87,8 +88,14 @@ generate_tarball_package() {
 
     # Copy other static package file(s).
     mkdir -p ${STG_DIR}/tarball/${tar_pkg_dir}/usr/sbin
+	
+    # Set AKS_USER variable to true inside aznfswatchdog to indicate use by Azure Kubernetes Service (AKS)
+    sed -i 's/AKS_USER=false/AKS_USER=true/' ${SOURCE_DIR}/src/aznfswatchdog
```

Comment:
Instead of changing the var in the original file, update the value once it is copied to specific location from where it will be included in tarball. 

In short, do this sed after L94 for "${STG_DIR}/tarball/${tar_pkg_dir}/usr/sbin/aznfswatchdog". Then you won't nedd L97 as well.